index.html

```diff
@@ -73,6 +73,7 @@ <h2 class="d2l-heading-3">Components</h2>
 			</ul>
 		</li>
 		<li><a href="components/expand-collapse/demo/expand-collapse-content.html">d2l-expand-collapse-content</a></li>
+		<li><a href="components/empty-state/demo/empty-state.html">d2l-empty-state</a></li>
```

Comment:
I'd flip this with expand/collapse to be alphabetical.

Reply:
Oops, thanks!